apps/webpack.js

```diff
@@ -80,6 +80,26 @@ if (envConstants.COVERAGE) {
   ];
 }
 
+var storybookConfig = _.extend({}, baseConfig, {
+  devtool: 'inline-source-map',
+  externals: {
+    "johnny-five": "var JohnnyFive",
+    "playground-io": "var PlaygroundIO",
+    "chrome-serialport": "var ChromeSerialport",
+    "blockly": "this Blockly",
+  },
+  plugins: [
+    new webpack.ProvidePlugin({React: 'react'}),
+    new webpack.DefinePlugin({
+      IN_UNIT_TEST: JSON.stringify(false),
+      'process.env.mocha_entry': JSON.stringify(process.env.mocha_entry),
+      'process.env.NODE_ENV': JSON.stringify(envConstants.NODE_ENV || 'development'),
+      BUILD_STYLEGUIDE: JSON.stringify(true),
+      PISKEL_DEVELOPMENT_MODE: false
```

Comment:
Technically this works just fine, but only because `JSON.stringify(bool)` is the same as `bool.toString()`.